analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantToCharArrayCall.CSharp11.cs

```diff
@@ -8,9 +8,9 @@ public class RedundantToCharArrayCall
     {
         public void CreateNew2(int propertyValue)
         {
```

Comment:
Please also add test cases where the invocation is on the ReadOnlySpan<T> variable and not the "UTF-8 string literal".